sys/include/net/nanocoap.h

```diff
@@ -548,28 +552,9 @@ static inline unsigned coap_get_id(const coap_pkt_t *pkt)
  */
 static inline unsigned coap_get_token_len(const coap_pkt_t *pkt)
 {
-    uint8_t tkl = pkt->hdr->ver_t_tkl & 0xf;
-
-    if (!IS_USED(MODULE_NANOCOAP_TOKEN_EXT)) {
-        return tkl;
-    }
```

Comment:
What happened with that?

Reply:
That is handled by `coap_hdr_get_token_len()`: If `nanocoap_token_ext` is not used, `coap_hdr_tkl_ext_len()` returns `0` no matter what, so `coap_hdr_get_token_len()` is only expensive when needed to be.